### PR TITLE
BBMap v38.44

### PR DESCRIPTION
**Pull request by peterk87**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bbmap/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% set name = "BBMap" %}
-{% set version = "38.22" %}
-{% set sha256 = "7ee6165440efc5fcfd108416d041e00c7d89002bc1fabfeb74a9c175528c9dea" %}
+{% set version = "38.44" %}
+{% set sha1 = "d66ba0802a191f5f63ef05148bb5bff6e09fcdd9" %}
+{% set md5 = "723c3d6a0d0a81aa4901c1d1e5e4d883" %}
 
 build:
   number: 1
```
Comment (dpryan79) [inline recipes/bbmap/meta.yaml]: Please reset the build number to 0

File: recipes/bbmap/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% set name = "BBMap" %}
-{% set version = "38.22" %}
-{% set sha256 = "7ee6165440efc5fcfd108416d041e00c7d89002bc1fabfeb74a9c175528c9dea" %}
+{% set version = "38.44" %}
+{% set sha1 = "d66ba0802a191f5f63ef05148bb5bff6e09fcdd9" %}
+{% set md5 = "723c3d6a0d0a81aa4901c1d1e5e4d883" %}
```
Comment (dpryan79) [inline recipes/bbmap/meta.yaml]: Just use one of the hashes both here and in the `source:` section.

Comment (peterk87) [reply recipes/bbmap/meta.yaml]: Okay! I wasn't sure which was preferred as SourceForge only had the sha1 and md5 checksums. Changed in e1c8fd00101d6150cffa605c16f87b98b504a3b0

File: recipes/bbmap/meta.yaml (hunk 1 above)
Comment (peterk87) [reply recipes/bbmap/meta.yaml]: Changed in e1c8fd00101d6150cffa605c16f87b98b504a3b0

**Comment by epruesse** [comment]
@peterk87 I'm closing this in favor of  #14560 - think I've got the JNI problem fixed.

**Comment by peterk87** [comment]
Thanks @epruesse! That's awesome 👍

**Pull request closed without merging**